subprojects/jvm-services/src/main/java/org/gradle/jvm/toolchain/internal/WindowsInstallationSupplier.java

```diff
@@ -80,8 +80,11 @@ private String getValue(String sdkSubkey, String path, String value, String vers
         return windowsRegistry.getStringValue(WindowsRegistry.Key.HKEY_LOCAL_MACHINE, sdkSubkey + '\\' + version + path, value);
     }
 
-    private List<String> findAdoptOpenJdk() {
-        return find("SOFTWARE\\AdoptOpenJDK\\JDK", "\\hotspot\\MSI", "Path");
+    private Stream<String> findAdoptOpenJdk() {
```

Comment:
It could be argued that Eclipse Adoptium/Temurin *is* AdoptOpenJDK since they are now the same project (if you try to download the latest JDK from AdoptOpenJDK, you get redirected to Temurin). However, I agree that these multiples names/renames are really confusing.

I have updated the method name to be `findOpenJDKs`. I think this is a much better name because it sidesteps this whole distraction. Modern OpenJDKs, no matter the vendor, are registering themselves under a similar registry key structure and this is what the method is finding.

I hope that you agree with this choice.

Reply:
I think the new name is fitting